charts/chainlink-cluster/values.yaml

```diff
@@ -197,7 +197,7 @@ opentelemetry-collector:
   mode: deployment
   image:
     repository: otel/opentelemetry-collector
-    tag: 0.81.2
+    tag: 0.95.0
```

Comment:
Should this be 0.82.0?

Reply:
we actually need the latest image because the the `0.82.0` has this bug:

https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/28553